packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md

```diff
@@ -40,6 +40,8 @@ TODO: Remove this section if there are not any general updates.
 
 * Improved Network profiler performance. [#7266](https://github.com/flutter/devtools/pull/7266)
 * Fixed a bug where selected pending requests weren't refreshing the tab once updated. [#7266](https://github.com/flutter/devtools/pull/7266)
+* Fixed the JSON viewer so multiline strings are visible in their row, and
+  through a tooltip [#7389](https://github.com/flutter/devtools/pull/7389)
```

Comment:
nit: period at end of sentence and separate PR links from description with a `-` character